### DISCUSSION TITLE
Drive Details dashboard missing data

### DISCUSSION THREAD

**Discussion by lq313c** [teslamate-org/teslamate#2819] [Q&A] [answered]
My Drive Details dashboard always shows "N/A" for the following panels, no matter what time range I select.
* Distance
* Duration
* Energy Used
* Odometer

 I can see there's data in the Drive, Tire Pressure, Elevation, and Temperatures panels, but nothing for the above.

Upon inspecting the panel queries, they all seem to be selecting `FROM drives d WHERE d.id = $drive_id`. This seems wrong. Shouldn't the query be calculating the relevant metric based on the total drives within the time range window, rather than a single specific `drive_id`? In fact, the panels seem unable to determine the `drive_id` at all, as query inspection shows `d.id = NULL`.

![image](https://user-images.githubusercontent.com/18491265/181688970-80bb017c-67df-44cf-925d-2637d1d3e8fc.png)

**Comment by DrMichael** [comment, marked as answer]
Drive Details is intended to be called from the Drives Dashboard for one specific drive. It is not intended to be called directly.

**Reply by lq313c** [reply]
Gotcha, that makes sense. I was using it as my home dashboard for a summary of the last 12hrs. Thanks!

**Reply by cwanja** [reply]
Would suggest using the Trips dashboard for that :)

**Comment by DrMichael** [comment]
Can we close this?
